src/frontend/csrs-portal/src/app/auth/auth-config.module.ts

```diff
@@ -0,0 +1,34 @@
+import { NgModule } from '@angular/core';
+import { AuthModule, LogLevel } from 'angular-auth-oidc-client';
+
+
+@NgModule({
+    imports: [AuthModule.forRoot({
+        config: {
+            triggerAuthorizationResultEvent: true,
+            postLoginRoute: '/welcomeuser',
+            logLevel: LogLevel.Debug,
+            historyCleanupOff: true,
+            eagerLoadAuthWellKnownEndpoints: false,
+            authority: 'https://dev.oidc.gov.bc.ca/auth/realms/onestopauth-basic',
+            redirectUrl: 'https://jag-csrs-portal-public-f0b5b6-dev.apps.silver.devops.gov.bc.ca',
```

Comment:
We can't hard code these values, they will vary by environment

Reply:
Yes, you are right! I am not going to hard code these values. I will develop to retrieve them from config file. The purpose for PR is to deploy on Openshift and check if it works.